avidemux_ru.ts

```diff
@@ -13890,7 +13891,8 @@ Only VP8/VP9 video and Vorbis/Opus audio supported</source>
         <location line="+0"/>
         <source>this mode has not been implemented
 </source>
-        <translation>этот режим ещё не разработан</translation>
+        <translation>этот режим ещё не разработан
+</translation>
```

Comment:
Definitely a bug in the source, fixed in the translation, now resurrected.

Reply:
Oh, i'll revert it in translation.